service/application/src/main/java/com/solace/maas/ep/event/management/agent/scanManager/rest/DataImportControllerImpl.java

```diff
@@ -34,17 +34,20 @@ public class DataImportControllerImpl implements DataImportController {
 
     private final ImportService importService;
 
+    private final Tracer trace;
```

Comment:
Added a tracer in order to get the traceId (which is done at line 50). The MDC.get was not working, so I got it from the current tracer context instead.